seed/static/seed/js/controllers/inventory_list_controller.js

```diff
@@ -721,6 +737,50 @@ angular.module('BE.seed.controller.inventory_list', [])
         });
       };
 
+      // evaluate all derived columns and add the results to the table
+      var evaluateDerivedColumns = function () {
```

Comment:
this could get out of hand for the inventory list... here we "batch" inventory IDs, then for every derived column evaluate that batch at once. Problem is that we aren't limiting concurrent requests here, we just fire them all off, not sure if that will cause problems down the line.

Reply:
I am following this approach and it does seem to work as expected. Maybe we just see how this reacts when stress tested? I suspect that it could definitely take some time to see the derived column values on inventory page load. 

I'm wondering the idea of sending each batch request independently and giving each request a resolve that updates the rows. I could see that possibly going badly but wanted to throw the idea out there.

Either way, this works and it's probably worth testing before changing. Just wanted to capture some thoughts.

Reply:
👍  yeah let's stress test it first
I would like to use async/await for batches but couldn't figure out how to make it work with some of the angular stuff